scripts/tapir/upgrade_coordinator.py

```diff
@@ -19,8 +19,13 @@ def main():
     deployer = Deployer.from_yaml(filepath=CONSTRUCTOR_PARAMS_FILEPATH, verify=VERIFY)
     instances = contracts_from_registry(filepath=ARTIFACTS_DIR / "tapir.json", chain_id=80002)
 
-    coordinator = deployer.upgrade(
-        project.Coordinator, instances[project.Coordinator.contract_type.name].address
+    implementation = deployer.deploy(project.Coordinator)
+    encoded_initializer_function = implementation.initializeNumberOfRituals.encode_input()
```

Comment:
I know functionally it is the same, but should we comment this out now that the first update is completed, like we did for `lynx`?

Reply:
hmm, good question, I don't know, I really used those lines, and each upgrade is individual

Reply:
Just to make sure I understand correctly. For the following lines:

```python
    encoded_initializer_function = implementation.initializeNumberOfRituals.encode_input()
    encoded_initializer_function = b""
```

The 1st is originally what you ran the upgrade script with, and now moving forward for any subsequent upgrade only the 2nd should be used...

Reply:
you are correct with one addition: 2nd would be only in case there are no other reinitializer functions that need to be called

Reply:
Right.

What about something like the following:
```suggestion
    # latest reinitializer function used for most recent upgrade - reinitializer(2)
    # encoded_initializer_function = implementation.initializeNumberOfRituals.encode_input()
```

Reply:
✔️